### PR TITLE
minor changes to toolbar style and other UI elements

### PR DESCRIPTION
**Pull request by dkanada**
I mainly made these changes to suggest that the icons in the dark theme be changed to white. This is the normal color for dark themes in other android apps, and the contrast in the dark theme is very bad at times in the HomeActivity toolbar. I also changed the navigation bar to a dark color so it wouldn't stand out as much on the dark themes. Here are some pictures below of before and after the color changes.

 

 

 

I also changed the hang up button in the call view to grey to match all the other buttons, it looked a bit odd in a different color to me.

### CODE REVIEW

**Comment by bmarty** [comment]
Hi @dkanada ,

Thanks for this new PR. Look good to me. You may be interested by doing what's in RiotAppCompatActivity: `TODO Maintenance: Toolbar is bound here now. Use this member in children Activities`, and to apply the up indicator in `configureToolbar` method as done in VectorRoomActivity.

Benoît

**Comment by dkanada** [comment]
Signed-off-by: Dillon Kanada <dkanada@protonmail.com>

**Comment by dkanada** [comment]
I could also change the colors in the bottom navigation bar on the home activity. I think it looks really nice. If nothing else, at least changing the icon colors on a fragment change would be a good improvement.

**Comment by bmarty** [comment]
I think it also fixes #2305 👍

**Comment by dkanada** [comment]
That should be everything. I removed the green navigation bar from the light theme, so now it is a light gray in the light theme and a dark gray in the dark theme. I also made a high resolution version of the play icon. I didn't want to redesign it since the android version is getting a redesign at some point. Might as well see what direction the iconography is going in first.

I remember at some point you guys were going to try adding custom themes to the mobile apps like the web app, is that still planned?

File: vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java [hunk 1]
```diff
@@ -740,12 +740,18 @@ private void loadVideo(final int position,
...
         final ImageView thumbView = view.findViewById(R.id.media_slider_video_thumbnail);
-        final ImageView playView = view.findViewById(R.id.media_slider_video_playView);
+        final ImageView playView = view.findViewById(R.id.media_slider_video_play);
 
         displayVideoThumbnail(view, !videoView.isPlaying());
 
+        videoView.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
+            @Override
+            public void onPrepared(MediaPlayer mp) {
+                mp.setLooping(true);
```
[the remark this answers is not in the document]
Comment (dkanada) [reply vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java]: I reverted this, but I actually think it is a great idea. Probably 90% of my videos are used as gifs which benefit from a looping format, and if they are long enough to be actual videos then a single loop every minute or so will not be too jarring of a user experience.

**Comment by dkanada** [comment]
Noticed a lot of dialogs in the app had odd padding, found the [issue](https://stackoverflow.com/questions/33074313/getting-default-padding-for-alertdialog) which affects any AlertDialog.Builder using the setView method. I will fix them as I come across them.

File: vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmarty) [reply vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java]: I understand this point of view, but maybe open a separate PR to introduce this change. In this PR it looks a bit offtopic :-)

**Pull request merged**